docs/cli-reference.md

```diff
@@ -1 +1,123 @@
-# CLI Reference
\ No newline at end of file
+# CLI Reference
+Dozer Command Line Interface is a unified tool and exposes commands for publishing and managing your applications, as well as lower level tools to interact with each component of your application. This reference guides you through getting started with the CLI.
+
+## Synopsis
+```
+dozer [options] <command> <subcommand> [parameters]
+```
+Use `dozer help`, `dozer -h` or `dozer --help` to view a list of available commands. Use `dozer <command> help`, `dozer <command> -h` or `dozer <command> --help` for information on a specific command with available subcommands. The synopsis for each command shows its parameters and their usage. Optional parameters are shown in square brackets.
+
+
+## Basic Commands
```

Comment:
I would distinguish between commands used in teh development lifecycle (like init, live, clean, etc) and commands used for deployments (like run). Mention that dozer has an app (pipeline) component and an api component and those can be run together or separately. Running together is generally used during development while running them separately is used during deployment. Then we point to a deployment page

Reply:
addressed